XamarinCommunityToolkit/XamarinCommunityToolkit.csproj

```diff
@@ -54,9 +54,6 @@
         <Compile Include="**/*.shared.cs" />
         <Compile Include="**/*.shared.*.cs" />
         <Compile Include="**/*.cs" />
-        <Compile Remove="obj\**" />
```

Comment:
@sthewissen Hi Steven, how are you getting this to build on your system without the <Compile Remove />?  On my system, it readds the obj folder to the solution.  I have to say "Exclude from project".  And it re-adds this <Compile Remove /> line.

Reply:
Not sure to be honest, I'm not doing anything special for it. I'm building/developing on macOS though?

Reply:
Yeah, I'm not sure if others are using Windows to build.  It doesn't build on my Windows system without the "Compile Remove" line.  I have tried removing that line multiple times, but it keeps re-adding the obj folder to the solution and throws thousands of errors.  If someone can try building this PR on Windows, that would be great.  I'll know I'm not going crazy.  :)  Thanks.